### ISSUE TITLE
Migration from google code wicked-charts-wicket7 to adesso wicked-charts-wicket8

### ISSUE THREAD

**Issue by sibiarunachalam** [adessoSE/wicked-charts#195]
Hi guys, 
We use google code wicked-charts-wicket7 for long time.
Now we want to upgrade to latest, so found new version in adesso wicked-charts-wicket8
But I couldn't use this as maven dependency, it is not found there.
I tried adding jcenter repository in maven setting.xml, it didn't work
Finally I was trying to download jar file for this dependency from https://oss.jfrog.org/artifactory/jcenter/. It asks credentials 
I tried with default credentials admin/password, it didn't work.
Could you help us to sort out the problem?

```
<repositories>
        <repository>
          <id>central</id>
          <url>http://central</url>
          <releases><enabled>true</enabled></releases>
        </repository>
        
        <repository>
          <snapshots><enabled>false</enabled></releases>     
          <id>jcenter</id>
          <name>jcenter</name>
          <url>https://oss.jfrog.org/artifactory/jcenter/</url>
        </repository>
      </repositories>
```

Development environment:
Java
Maven
Spring
Wicket
Highcharts

**Comment by maximAtanasov** [comment]
Hello, which version of Wicked-Charts are you using? Currently, only versions 3.1.0, 3.2.0 and 3.2.1 are available in jCenter.

**Comment by sibiarunachalam** [comment]
This is currently we use.
```
 <dependency>
                <groupId>com.googlecode.wicked-charts</groupId>
                <artifactId>wicked-charts-wicket7</artifactId>
                <version>2.0.1</version>
            </dependency>
```

Now upgrading to 
```
 <dependency>
                <groupId>de.adesso.wicked-charts</groupId>
                <artifactId>wicked-charts-wicket8</artifactId>
                <version>3.2.1</version>
            </dependency>
```
Yes, it is available in jCenter
but wicked-charts-wicket8 is not resolved in maven project even after configuring the repository in maven setting.xml
I couldn't even download it manually.
Please correct me If I did any mistake.

**Comment by maximAtanasov** [comment]
Yes, you are correct and jCenter asks for credentials. This apparently has to do with the shutdown of jCenter. According to jfrog however, the repository should be available as read-only indefinetly, so I don't know exactly what credentials it needs either. Until the library is migrated to maven central, the only way to download it is through the GitHub releases, which is quite inconvenient. I will attempt to find out if jCenter can still be used.
